controllers/utils.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// extractNodeSelectorAndTolerations combines and extracts scheduling parameters from the multiple deviceClass entries in an lvmCluster
+func extractNodeSelectorAndTolerations(lvmCluster lvmv1alpha1.LVMCluster) (*corev1.NodeSelector, []corev1.Toleration) {
```

Comment:
This function is also needed in the topolvm node. We can add a second PR to do the link once we have merged this ( i will take a look to it)

Reply:
No need - it will be picked up automatically as it is in the same package.

Reply:
we will need to change it in "topolvm node" , because the function is not available in the repo, ( for the moment none of the prs with the function has been merged)

Reply:
The PR removes the `extractNodeSelectorAndTolerations` from the `topolvm_node.go` file.  There is an existing call to `extractNodeSelectorAndTolerations` in the `topolvm_node.go`. So that will refer to this function.